service/src/main/java/io/camunda/service/UsageMetricsServices.java

```diff
@@ -33,18 +37,22 @@ public UsageMetricsServices(
   }
 
   @Override
-  public SearchQueryResult<UsageMetricStatisticsEntity> search(final UsageMetricsQuery query) {
+  public SearchQueryResult<Tuple<UsageMetricStatisticsEntity, UsageMetricTUStatisticsEntity>>
+      search(final UsageMetricsQuery query) {
     if (query == null) {
       throw new IllegalArgumentException("Query must not be null");
     }
     validateStartAndEndTime(query);
-    return SearchQueryResult.of(
+    final UsageMetricsSearchClient authUsageMetricsSearchClient =
         usageMetricsSearchClient
             // TODO add proper authorization with https://github.com/camunda/camunda/issues/34708
             .withSecurityContext(
-                securityContextProvider.provideSecurityContext(
-                    authentication, Authorization.of(a -> a)))
-            .usageMetricStatistics(query));
+            securityContextProvider.provideSecurityContext(
+                authentication, Authorization.of(a -> a)));
+    return SearchQueryResult.of(
+        Tuple.of(
+            authUsageMetricsSearchClient.usageMetricStatistics(query),
+            authUsageMetricsSearchClient.usageMetricTUStatistics(query)));
```

Comment:
💭 It would be nice to execute the two queries in parallel somehow.

I'm not sure right now if something like this already exists. If it doesn't exist yet and complicated then we can have a follow-up ticket for this.